app/index.html

```diff
@@ -8,15 +8,25 @@
     <meta name="theme-color" content="#263947">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    {{content-for "head"}}
+      {{content-for "head"}}
 
     <!-- All CSS should be included using sass via app.scss -->
     <!-- We have no control over what 3rd parties import into vendor.css -->
     <!-- <link rel="stylesheet" href="{{rootURL}}assets/vendor.css"> -->
     <link rel="stylesheet" href="{{rootURL}}assets/reviews.css">
     <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,600,300,700' rel='stylesheet' type='text/css'>
 
-    {{content-for "head-footer"}}
+      {{content-for "head-footer"}}
+
+      <script type="text/javascript"
```

Comment:
This should be marked with `async` as described on [MathJax's docs](http://docs.mathjax.org/en/latest/start.html). Otherwise the entire page will have to wait for mathjax to initialize.